frontend/projects/upgrade/src/app/features/dashboard/home/components/modal/export-experiment/export-experiment.component.ts

```diff
@@ -75,17 +75,17 @@ export class ExportModalComponent implements OnInit {
     this.experimentService.exportExperimentInfo(experimentId, experimentName);
     this.openSnackBar(true);
   }
-  exportExperimentDesign(experimentId: string) {
-    this.experimentService.exportExperimentDesign(experimentId);
+  exportExperimentDesign(experimentIds: string[]) {
+    this.experimentService.exportExperimentDesign(experimentIds);
     this.openSnackBar(false);
   }
 
   exportExperiment() {
     const { exportMethod } = this.exportForm.value;
     if (exportMethod == EXPORT_METHOD.DATA) {
-      this.exportExperimentInfo(this.experiment.id, this.experiment.name)
+      this.exportExperimentInfo(this.experiments[0].id, this.experiments[0].name)
```

Comment:
what if `experiments[0]` is undefined? Just found out today that typescript won't warn us in this kind of spot, and it has caused us runtime errors.

Reply:
Put another check in if statement for experiment[0]